ls1intum/Artemis
# `Development`: Remove participation status in client

### Checklist
#### General


#### Client

### Motivation and Context


#5791

### Description

Remove the ParticipationStatus and replace it with the corresponding, often simpler conditions in the client code

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Programming Exercise
- 1 Modeling/File Upload/Text Exercise
- 1 Quiz Exercise (synchronized mode)
- 1 Quiz Exercise (batch mode)

1. Log in to Artemis
2. Particpate in all exercises as student and check that everything (starting, submitting, reviewing result, ...) still works

#### Exam Mode Testing



Prerequisites:
- 1 Instructor
- 2 Students
- 1 Exam with each exercise types

1. Log in to Artemis
2. Participate in the exam as a student
3. Make sure that a test run still works
4. Make sure that the UI of all exercises in the exam mode stays unchanged. You can use the [exam mode documentation](https://docs.artemis.ase.in.tum.de/user/exam_mode/) as reference.

### Review Progress






#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage
